packages/frontend/src/components/wallet/Wallet.js

```diff
@@ -321,7 +320,6 @@ export function Wallet({
         <StyledContainer
             className={SHOW_NETWORK_BANNER ? 'showing-banner' : ''}
         >
-            <ReleaseNotesModal />
```

Comment:
Not sure we need to do a more in-depth removal given that the component may have some future potential.